admin/class-admin-asset-manager.php

```diff
@@ -263,9 +268,9 @@ protected function scripts_to_be_registered() {
 					'jquery',
 					'jquery-ui-core',
 					'jquery-ui-progressbar',
+					self::PREFIX . 'react-dependencies',
 					self::PREFIX . 'select2',
 					self::PREFIX . 'select2-translations',
-					self::PREFIX . 'react-dependencies',
```

Comment:
React dependancies aka the webpack main bundle must be loaded first to avoid issues. see[ this article](https://medium.com/@MarkEwersDev/note-to-self-if-you-ever-get-this-uncaught-referenceerror-webpackjsonp-is-not-defined-message-and-d354f5c4d335) for more context